config/file.go

```diff
@@ -202,6 +212,10 @@ func (fs *FileStore) HasFile(name string) (bool, error) {
 
 // RemoveFile removes a previously persisted configuration file.
 func (fs *FileStore) RemoveFile(name string) error {
+	if filepath.IsAbs(name) {
+		// Don't delete absolute filenames, as may be mounted drive, etc.
```

Comment:
Worth adding a debug log in this case, just in case our assumptions here are wrong and it catches a customer by surprise?

Reply:
Agreed